sandbox/src/main/java/com/bumble/appyx/sandbox/client/workflow/WorkflowExampleActivity.kt

```diff
@@ -0,0 +1,92 @@
+package com.bumble.appyx.sandbox.client.workflow
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.layout.Column
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.lifecycle.lifecycleScope
+import com.bumble.appyx.core.integration.NodeHost
+import com.bumble.appyx.core.integrationpoint.NodeActivity
+import com.bumble.appyx.core.plugin.ParenNodeBuilt
+import com.bumble.appyx.sandbox.ui.AppyxSandboxTheme
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.launch
+
+class WorkflowExampleActivity : NodeActivity() {
+
+    lateinit var rootNode: RootNode
+
+    private var job: Job? = null
+
+    fun handleDeepLink(intent: Intent) {
+        if (intent.action == Intent.ACTION_VIEW) {
+            job?.cancel()
+            job = workflowFactory.invoke(intent)
+        }
+    }
+
+    override fun onDestroy() {
+        super.onDestroy()
+        job?.cancel()
+    }
+
+    private val workflowFactory: (Intent) -> Job? = {
+        when {
+            // adb shell am start -a "android.intent.action.VIEW" -d "appyx-sample://workflow1"
```

Comment:
These would be a good candidate for androidTests

Reply:
Yes, tests will be added once we agree on the api